docs/sdkx-unity/getting-started-android.mdx

```diff
@@ -99,15 +99,15 @@ The `mainTemplate.gradle` is genereted when you enable the **Custom Gradle Templ
 The `build.gradle` exists in generated Gradle project when you enable the **Export Project** property on the Player window and Build the project.
 
 Update **dependencies** section of the `mainTemplate.gradle` or `build.gradle` file as:
-~~~code
+
+```
```

Comment:
I hope not specifying a language defaults to a UI that's not broken.

Reply:
Yes, it shows a normal code block without syntax highlighting